source/cpp/source/DefaultCommandHandler.cpp

```diff
@@ -136,7 +136,7 @@ Response keyPress (const Command & command)
 
     auto modifiers = command.getArgument ("modifiers");
     auto componentId = command.getArgument ("focus-component");
-    auto windowId = command.getArgument (ComponentSearch::windowId);
+    auto windowId = command.getArgument ("window-id");
```

Comment:
I imagine you removed the `windowId` constant from `ComponentSearch` because you wanted it to just be an implementation detail but I'm not sure how I feel about this string being repeated all over this file. Maybe at the least there could be a local constant in this cpp for it?

Reply:
Yeah, I'll change this. Sometimes I feel when there are lots of string arguments it's easier to have them hardcoded. If the value changes, it's still only a find and replace. But... I guess with IDE refactoring it would be more of a pain.